cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreServiceUtils.java

```diff
@@ -496,4 +511,35 @@ private static File updateHiveConfFile(File confFile, File tempDir) {
     }
     return newHiveConfFile;
   }
+
+  public static boolean isTezAvailable() {
```

Comment:
If `System.getenv(Constants.TEZ_HOME)` is not available on explore container, this method should be made private. And also add a note to this method.